[2025-09-08T17:04:54.521+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-08T17:04:54.531+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskC scheduled__2025-08-30T12:00:00+00:00 [queued]>
[2025-09-08T17:04:54.536+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskC scheduled__2025-08-30T12:00:00+00:00 [queued]>
[2025-09-08T17:04:54.536+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-09-08T17:04:54.550+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): taskC> on 2025-08-30 12:00:00+00:00
[2025-09-08T17:04:54.556+0000] {standard_task_runner.py:72} INFO - Started process 17835 to run task
[2025-09-08T17:04:54.559+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'cron_catchup_backfill', 'taskC', 'scheduled__2025-08-30T12:00:00+00:00', '--job-id', '143', '--raw', '--subdir', 'DAGS_FOLDER/executing_cron_catchup_backfill.py', '--cfg-path', '/tmp/tmpfzjbos4k']
[2025-09-08T17:04:54.562+0000] {standard_task_runner.py:105} INFO - Job 143: Subtask taskC
[2025-09-08T17:04:54.601+0000] {task_command.py:467} INFO - Running <TaskInstance: cron_catchup_backfill.taskC scheduled__2025-08-30T12:00:00+00:00 [running]> on host codespaces-9bd7b3
[2025-09-08T17:04:54.648+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='loonycorn' AIRFLOW_CTX_DAG_ID='cron_catchup_backfill' AIRFLOW_CTX_TASK_ID='taskC' AIRFLOW_CTX_EXECUTION_DATE='2025-08-30T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-08-30T12:00:00+00:00'
[2025-09-08T17:04:54.648+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-08T17:04:54.659+0000] {logging_mixin.py:190} INFO - TASK C executed!
[2025-09-08T17:04:54.659+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-09-08T17:04:54.664+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-08T17:04:54.664+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cron_catchup_backfill, task_id=taskC, run_id=scheduled__2025-08-30T12:00:00+00:00, execution_date=20250830T120000, start_date=20250908T170454, end_date=20250908T170454
[2025-09-08T17:04:54.698+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-08T17:04:54.710+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-08T17:04:54.714+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
